source/Straight_hand.py

```diff
@@ -0,0 +1,39 @@
+from CardGames import *
+from Find_Flush import *
+
+
+def straight(player, river):
```

Comment:
There are some more efficient ways of doing this, but this is good and it works.